setup.py

```diff
@@ -38,6 +38,10 @@
         "setuptools",
         "docutils",
         "pygments",
+        "requests"
```

Comment:
```suggestion
        "requests",
```

To avoid future syntax errors and extra diff noise (unless you're planning to run black when this is ready)